src/templates/appcodetails.js

```diff
@@ -369,6 +369,7 @@ class AppDetails extends Component {
             object: this.props.data.apps.website,
           })
     await comment.save()
+    // await comment.insertOne()
```

Comment:
Please open a new PR if you want to address the issue of deprecated `save` method.

Reply:
okay sure